diff --git a/node_modules/@protontech/pmcrypto/.bun-tag-510b298809627b34 b/.bun-tag-510b298809627b34
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/@protontech/pmcrypto/.bun-tag-bbd8917749136bea b/.bun-tag-bbd8917749136bea
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/lib/openpgp.ts b/lib/openpgp.ts
index 134ae5bae14dc0c9241f7122f6f5d033a9e9ca60..3b2b952b0da0168112b4cc16fad9dacd72544e16 100644
--- a/lib/openpgp.ts
+++ b/lib/openpgp.ts
@@ -1,4 +1,4 @@
-import { config } from 'openpgp/lightweight';
+import { config } from 'openpgp';
 
 export const setConfig = () => {
     config.s2kIterationCountByte = 255;
@@ -31,4 +31,4 @@ export const setConfig = () => {
     config.enableParsingV5Entities = true;
 };
 
-export * from 'openpgp/lightweight';
+export * from 'openpgp';
diff --git a/lib/pmcrypto.d.ts b/lib/pmcrypto.d.ts
index 49868bbe9f0b8e4e254c18bfb0992bab47deb54c..dd8e98695375e4d9b90e98a0bc0519e28c456089 100644
--- a/lib/pmcrypto.d.ts
+++ b/lib/pmcrypto.d.ts
@@ -19,7 +19,7 @@ import {
     readKey, readKeys, readPrivateKey, readPrivateKeys,
     type PartialConfig,
     type AlgorithmInfo
-} from 'openpgp/lightweight';
+} from 'openpgp';
 
 import { VERIFICATION_STATUS, SIGNATURE_TYPES } from './constants';
 import type { ContextSigningOptions, ContextVerificationOptions } from './message/context';
